Dz 04

chicony83 opened the pull request.

Commit:
add 3 buttons

Commit:
add DZ1Aktivity

Commit:
add listener onButton MainAktivity

Commit:
rename textView in second DZ1Activity

Commit:
update DZ2

Commit:
Add objekts to aktivity_d_z1

Commit:
DZ1Aktivity {
   add onClickListener on button
}
aktivity_d_z1{
   moving elements
}

strings{
  adding strings
  deleting strings
}

Commit:
move strings in gradle

Commit:
add DZ4HarvestingAktivity.kt
add background in aktivity_main.xml

Commit:
add in MainActivity.kt setOnClickListener to button
rename Button in activity_main.xml
add strings
modify AndroidManifest.xml

Commit:
add in activity_dz_4_harvesting.xml{
background
add title
add 3 constraintlayouts
}
add alpha color in colors.xml
add strings
add style
modify AndroidManifest

Commit:
add names to any company
add strings
add styles to styles

Commit:
add containers for agrikultures

add styles to styles

Commit:
add agricultures in Company1 in activity_dz_4
add color
add styles to styles

Commit:
worked!!!

Commit:
add Winner

stasenkots reviewed (changes requested):
You should use ViewModel in the task.And coroutines for emulating "receiving some data from server".also better to group repeated logic

Commit:
edit delay
edit toast

Commit:
mod DZ4HarvestingAktivity.kt

create IncomingData.kt {ViewMode}
create Random.kt {return Int (0..8).random}
create TextView.kt

Commit:
working with one variable MVVM

in DZ4HarvestingAktivity.kt
adding observer at one variable

in IncomingData.kt
adding new methods getData and postValue

Commit:
working MVVM

in DZ4HarvestingAktivity.kt
adding observer at all variable

in IncomingData.kt
adding all variables in methods getData and postValue

stasenkots commented on app/src/main/java/com/example/myhomeworks/IncomingData.kt:
```diff
@@ -0,0 +1,72 @@
+
+    var winner = false;
+
+    var company1_potato = MutableLiveData(0)
```
Repeated logic.You can keep vegitables in some class.
